documentation/src/main/markdown/troubleshooting/commonwindowsintricacies.md

```diff
@@ -0,0 +1,4 @@
+# Windows Intricacies
```

Comment:
Suggestion - name the file something basic like windowsos.md as it will already reside under ../troubleshooting
Page title:
# Windows OS 
Then use sub titles under that to break the information into sections.
Such as:
## Passing spaces in arguments